<?xml version="1.0"?>
<launch>
  <group>
    <push-ros-namespace namespace="sensing/lidar/top"/>

    <node pkg="fog_reduction" exec="fog_reduction" name="fog_reduction" output="screen">
      <remap from="input" to="pointcloud_raw_ex"/>
      <remap from="output" to="fog_reduction/pointcloud_ex"/>
      <param name="intensity_threshold" value="5"/>
      <param name="density_threshold" value="120"/>
      <param name="density_radius" value="1.4"/>
      <param name="use_sim_time" value="true" />
    </node>
  </group>

</launch> 